hiera.tf

```diff
@@ -0,0 +1,14 @@
+module "hiera" {
```

Comment:
As mentioned in https://github.com/appuio/terraform-modules/pull/1, this module should not be instantiated here, instead it should be instantiated in the "lb-cloudscale" module, so that it's sufficient to instantiate module "lb-cloudscale" in this module to get working LB VMs.

Reply:
I did do it because of the dependency cycle between needing lb ips in `hosts.allow` and needing worker ips in `lb-cloudscale`. With the removal of `hosts.allow` it can be merged again